includes/class-wc-payments-webhook-reliability-service.php

```diff
@@ -155,9 +157,17 @@ private function schedule_process_event( string $event_id ) {
 	/**
 	 * Schedule a job to fetch failed events.
 	 *
+	 * We will bail if this is called too early and ActionScheduler is not initialized.
+	 *
 	 * @return void
 	 */
 	private function schedule_fetch_events() {
+		// If the ActionScheduler is not initialized, we will get a fatal error when trying to schedule.
+		if ( ! ActionScheduler::is_initialized() ) {
+			Logger::info( 'Skipping scheduling a job to fetch failed events from the server: ActionScheduler is not initialized.' );
```

Comment:
The thing is it is not an error or something we should investigate - that is why I went with info. Simply, we refresh the account cache too early for ActionScheduler jobs to be attached based on it. We can't do anything about it.

WDYT?

Reply:
Ah, ok then. I was triggered by fatal error before and now we just making it info and I thought it can be fixed by the merchant
Thanks for explanation!